app/developers/templates/developers/collection_tasklist.html

```diff
@@ -74,18 +71,18 @@ <h2 class="govuk-heading-m">{{ section.title }}</h2>
       {% endfor %}
     </div>
   </div>
-  {% if collection_helper.status != statuses.COMPLETED %}
+  {% if submission_helper.status != statuses.COMPLETED %}
```

Comment:
Doesn't need to happen here but just to remind myself:
```suggestion
  {% if not submission_helper.is_completed %}
```